examples/kv/ids/kv_and_py/kv_and_py.py

```diff
@@ -0,0 +1,33 @@
+'''
+ Referring on ids from .py
+==================================
+
+This example shows how to refer to an id from a .py file.
+'''
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+
+import kivy
+kivy.require('1.8.0')
+
+
+class RootWidget(BoxLayout):
+
+    def first_function(self, status):
+        # print out the given parameter
+        print(status)
+        # check the status of the switch by referring on the id
+        if self.ids['my_switch'].active is True:
```

Comment:
You could do `self.ids.my_switch.active` instead, so it conforms with the rest of the examples and docs.